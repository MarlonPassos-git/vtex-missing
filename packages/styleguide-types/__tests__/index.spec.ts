import type { Props } from '../src/Alert';
import { testType } from './test-utils';

// @dts-jest:group Alert
{
  // @dts-jest:pass:snap
  testType<Props>({
    children: 'children',
    type: 'success',
    action: {
      label: 'label',
      onClick: () => {}
    },
    autoClose: 1000,
    closeIconLabel: 'closeIconLabel',
    focusOnOpen: true,
    onClose: () => {}
  });
}
