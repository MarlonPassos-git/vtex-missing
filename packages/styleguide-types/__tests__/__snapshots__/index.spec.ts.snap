// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert testType<Props>({
      children: "children",
      type: "success",
      action: {
        label: "label",
        onClick: () => { }
      },
      autoClose: 1000,
      closeIconLabel: "closeIconLabel",
      focusOnOpen: true,
      onClose: () => { } 
    }) (type) should match snapshot 1`] = `"Props"`;
